src/lectern-client/index.ts

```diff
@@ -1,5 +1,5 @@
-import * as schemaManager from '../submission/schema/schema-manager';
```

Comment:
I dont think this is used? lectern client should not have any dependency except for utils

Reply:
correct, this was already there for some reason. will remove :+1: